### PR TITLE
Added more info to README, included writeup template, changed drive.py to accept model.h5 file

### PR DESCRIPTION
**Pull request by andrewpaster**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,71 @@
...
+
+### Dependencies
+This lab requires:
+
+* [CarND Term1 Starter Kit](https://github.com/udacity/CarND-Term1-Starter-Kit)
+
+The lab enviroment can be created with CarND Term1 Starter Kit. Click [here](https://github.com/udacity/CarND-Term1-Starter-Kit/blob/master/README.md) for the details.
+
+The following resources can be found in this github repository:
+* simulator
```
Comment (domluna) [inline README.md]: Isn't the simulator is downloaded separately from the classroom?

Comment (andrewpaster) [inline README.md]: misunderstood. Thought the simulator was going to be added to the repository. It's still going to be downloaded from the classroom?

Comment (domluna) [inline README.md]: Yep, still from the classroom. I don't see a good reason to change that. An option could be to add it to the releases tab but that's basically the same as having it in the classroom IMO.

**Comment by domluna** [comment]
LGTM 👍

**Pull request merged**